src/bindings/cpp/include/keyset.hpp

```diff
@@ -31,7 +31,9 @@ class KeySet
 	inline KeySet(ckdb::KeySet *k);
 	inline KeySet(const KeySet &other);
 
-	inline explicit KeySet(size_t alloc, va_list ap);
+#if defined(__amd64__) || defined(__x86_64__) || defined(__i386__)
```

Comment:
I have no problem with removing the va constructor. Shall I remove it right away and rebase?